src/lexer/lexer.rs

```diff
@@ -404,6 +495,10 @@ impl Lexer {
         self.source[self.current..].chars().nth(1)
     }
 
+    fn triple_peek(&self) -> Option<char> {
+        self.source[self.current..].chars().nth(2)
+    }
```

Comment:
In future maybe we can add a method which takes peek number

Reply:
I think this is the max peek we might need based on the python tokens present now. So probably we won't need another peek_4